### PR TITLE
Nagra

### PR DESCRIPTION
**Pull request by RothIOP**
Fixes issue #12.

### CODE REVIEW

File: custom_modules/URLQueryBuilder.js [hunk 1]
```diff
@@ -163,10 +163,11 @@ module.exports.URLQueryBuilder = class {
...
         this._nodeArray = this.urlToGraph(this._url)
         let nodeMap = this._nodeArray
-        let query = 'UPDATE'
-        let primaryClauses = ''
-        let subClauses = ''
-
+        let query = 'INSERT INTO'
+        let columns = ''
+        let values = ''
+        let pk = ''
```
Comment (danielc2013) [inline custom_modules/URLQueryBuilder.js]: what is pk? primaryKey? There's no need to conserve characters, make sure to use verbose variable names. We have others who are new to the concept and we need it to be self documenting as much as possible

File: custom_modules/URLQueryBuilder.js [hunk 2]
```diff
@@ -123,7 +123,7 @@ module.exports.URLQueryBuilder = class {
             // Primary Clauses
             if(node.primaryKey){
                 primaryClauses += ` ${node['name']}.${this._configMap[node['name']].fields.primary_key} = '${node.primaryKey}'`
-                if(nodeIndex < nodeMap.length - 1)
+                if(nodeIndex < nodeMap.length - 2)
```
Comment (danielc2013) [inline custom_modules/URLQueryBuilder.js]: This is fine for now, but we need to start looking at writing newman tests. Because something tells me this'll break in edge cases
